### PR TITLE
Vulnerability causing pipeline to fail

### PR DESCRIPTION
**Pull request by gms-gs**
## Context

Pipeline is failing because of [this vulnerability](https://security.snyk.io/vuln/SNYK-ALPINE320-AOM-7578253) (aom) identified by Snyk.

Here is a [failed run](https://github.com/DFE-Digital/apply-for-teacher-training/actions/runs/10244729214/job/28341720276)
Fix: Upgrade Alpine:3.20 aom to version 3.9.1-r0 or higher.

## Changes proposed in this pull request

Bump AOM version

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)
- [ ] Inform data insights team due to database changes
- [ ] Make sure all information from the Trello card is in here
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Add PR link to Trello card

### CODE REVIEW

**Comment by inulty-dfe** [comment]
@gms-gs Can we update the PR description by removing the irrelevant references to graphviz etc

**Review by CatalinVoineag — APPROVED** [review]
Nice

**Review by dcyoung-dev — APPROVED** [review]

**Review by inulty-dfe — APPROVED** [review]
Nice work :+1:

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,8 @@ FROM ${BASE_RUBY_IMAGE} AS gems-node-modules
 
 RUN apk -U upgrade && \
     apk add --update --no-cache git gcc libc-dev make postgresql-dev build-base \
-    libxml2-dev libxslt-dev nodejs yarn tzdata libpq libxml2 libxslt graphviz chromium gcompat
+    libxml2-dev libxslt-dev nodejs yarn tzdata libpq libxml2 libxslt graphviz chromium gcompat \
+    'aom>=3.9.1-r0'
```
Comment (elceebee) [inline Dockerfile:10]: 👍

**Review by elceebee — APPROVED** [review]

**Pull request merged**